resources/main.mjs

```diff
@@ -260,12 +265,59 @@ class MainBenchmarkClient {
         return JSON.stringify(this._measuredValuesList, undefined, indent);
     }
 
+    _getFormattedCSVResult() {
+        let tests = [];
+        // The CSV format is similar to the details view table. Each measurement is a row with
+        // the name and N columns with the measurement for each iteration:
+        // ```
+        // Measurement,#1,...,#N
+        // TodoMVC-JavaScript-ES5-Total,num,...,num
+        // TodoMVC-JavaScript-ES5-Adding100Items,num,...,num
+        // TodoMVC-JavaScript-ES5-Adding100Items-Sync,num,...,num
+        // TodoMVC-JavaScript-ES5-Adding100Items-Async,num,...,num
+        // ...
+        // TodoMVC-JavaScript-ES6-Total,num,...,num
+        // TodoMVC-JavaScript-ES6-Adding100Items,num,...,num
+        // TodoMVC-JavaScript-ES6-Adding100Items-Sync,num,...,num
+        // TodoMVC-JavaScript-ES6-Adding100Items-Async,num,...,num
+        // ```
+        for (let i in this._measuredValuesList[0].tests) {
+            tests.push([`${i}-Total`]);
+            for (let j in this._measuredValuesList[0].tests[i].tests) {
```

Comment:
Please don't use one letter variable like "j".
A more appropriate variable name would be "testName" here.